pages/common/xml-validate.md

```diff
@@ -0,0 +1,24 @@
+# xml-validate
+
+> Validate XML documents.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Validate XML document for well-formedness only (default):
```

Comment:
Perhaps this will read better?
```suggestion
- Validate an XML document for well-formedness only (default behaviour):
```

Reply:
- Validate XML document(s) for well-formedness only (default behavior):

Reply:
- Validate XML document(s) for well-formedness only:

`xml validate {{path/to/input1.xml|URI}} {{input2.xml ...}}`